src/app/landing/LandingPage.jsx

```diff
@@ -65,26 +65,36 @@ class LandingPage extends Component {
               <source
                 className="OnboardingImage"
                 media="(min-aspect-ratio: 2/3)"
-                srcSet="img/landing_page/map_tablet.png"
+                srcSet="img/landing_page/step1.png"
               />
               <img
                 className="OnboardingImage"
-                src="img/landing_page/map_narrow.png"
+                src="img/landing_page/step1_mobile.png"
                 alt="map showing food, clothing, and personal care markers"
               />
             </picture>
           </div>
 
           <div className="Section">
             <div className="OnboardingSectionNumber">2</div>
-
             <div className="OnboardingText">
               <h2>Find providers that will actually serve <span className="Emphasis">you</span></h2>
               <p className="text-lighter">
                 Filter or answer a few questions and we will show you locations that meet your specific needs, from age to gender to family situations.
               </p>
             </div>
-            <img className="OnboardingImage" src="img/landing_page/welcome.jpg" alt="" />
+            <picture>
+              <source
+                className="OnboardingImage"
+                media="(min-aspect-ratio: 2/3)"
+                srcSet="img/landing_page/step2.png"
```

Comment:
One of the other notes highlighted red in the web.dev audit was that we should be using "next-gen formats" for these pictures, rather than pngs and jpgs. Not necessary to do it as part of this PR, but since we're already replacing these assets, maybe it would be simpler?